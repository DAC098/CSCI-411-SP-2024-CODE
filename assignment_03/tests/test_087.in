7 10
17 57 188 199 434 462 553
7254
5229
9304
652
481
1843
3483
4417
9270
8004
