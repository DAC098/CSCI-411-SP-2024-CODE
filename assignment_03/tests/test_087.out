1 0 1 0 2 5 7 
0 0 0 0 1 2 7 
0 0 2 1 1 0 15 
35 1 0 0 0 0 0 
0 0 0 0 0 0 0 
3 0 0 0 2 2 0 
0 1 0 1 0 1 5 
0 1 1 0 6 1 2 
3 0 0 0 0 2 15 
1 1 1 0 0 0 14 
